ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -387,7 +387,7 @@ ORK_CLASS_AVAILABLE
  The text for each answer is given more prominence than the `detailText` in the row, but
  both are shown.
  */
-@property (readonly, copy) NSArray *textChoices;
+@property (readonly, strong) NSArray *textChoices;
```

Comment:
This is intentionally copy, although it's irrelevant as API because it's a readonly property. Do not change.